docs/src/Switch.doc.js

```diff
@@ -55,6 +56,30 @@ card(
   />,
 );
 
+card(
+  <MainSection name="Usage guidelines">
+    <MainSection.Subsection columns={2}>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        title="When to Use"
+        description={`
+          - For a binary option that can be either active or inactive.
+          - Typically used on mobile, where toggling the Switch takes immediate effect.
+        `}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        title="When Not to Use"
+        description={`
+          - Choosing between multiple, related options. Use [Checkboxes](Checkbox) or [RadioButtons](/RadioButton) instead.
```

Comment:
This one is kinda confusing, i don't think it adds much and could led to confusion if you need +1 switches together for any reason.

Reply:
what do you think of the updated one?